lib/git.zsh

```diff
@@ -77,8 +77,8 @@ function git_current_branch() {
 # Gets the number of commits ahead from remote
 function git_commits_ahead() {
   if command git rev-parse --git-dir &>/dev/null; then
-    local commits="$(git rev-list --count @{upstream}..HEAD)"
-    if [[ "$commits" != 0 ]]; then
+    local commits="$(git rev-list --count @{upstream}..HEAD 2>/dev/null)"
+    if [[ -n $commits ]] && [[ "$commits" != 0 ]]; then
```

Comment:
Two issues:
1. You should **always** soround any environment variable with '"' when testing, otherwise you might accidently execute:
```[[ -n ]]``` which is an syntax error...
1. Why are you using two tests and combing there results instead of using just a single test testing both conditions?
```[[ -n "$commits" -a "$commits" != "0" ]]``` or
```[[ -n "$commits" -a $commits -ne 0 ]]```
Don't know which one is faster ;)

Reply:
Btw.:
```[[ "$commits" != "0" ]]``` alone just works fine for this case ;)